angularjs-portal-frame/src/main/webapp/main.js

```diff
@@ -0,0 +1,59 @@
+require.config({
+
+  paths: {
+    'my-app': 'my-app.js'
+    /* TODO: Incorporate the <scripts/> that are being globally imported in frame.html

```

Comment:
I'm with @paulerickson here, small bites :)

I think that can come with the next iteration.